dir.targets

```diff
@@ -56,4 +56,11 @@
     </SupplementalTestData>
   </ItemGroup>
 
+  <Target Name="ProducesPackageId" 
```

Comment:
I don't think we'd want to teach RepoUtils about our build system.  If we decide that the filtering logic which is intrinsic to our build isn't a necessary part of the equation (seems probable), then we could take a slightly different tact and just scour the tree for pkgproj's to glean information from.  In that case, adding this to RepoUtils would be more palatable.